.github/workflows/build-push.yml

```diff
@@ -27,6 +28,29 @@ jobs:
 
     # Executed sequentially when job runs
     steps:
+      # Check and ensure that the repository has all the variables and secrets set
+      - name: Check User Set Variables
+        run: |
+          if [[ -z "$DOCKER_USER" ]]; then \
+          echo "::error::Secret DOCKER_USER was not set"; \
+          exit 1; \
+          fi
+          if [[ -z "$DOCKER_TOKEN" ]]; then \
+          echo "::error::Secret DOCKER_TOKEN was not set"; \
+          exit 1; \
+          fi
+          if [[ -z "$DOCKER_REGISTRY" ]]; then \
+          echo "::error::Variable DOCKERHUB_REPO was not set"; \
```

Comment:
This error message should be changed to `DOCKER_REGISTRY` not `DOCKERHUB_REPO`

Reply:
Talked about in 1:1 meeting, decided no need to change.